### DISCUSSION TITLE
How to use the bigint.unsigned.handling.mode parameter in a Paimon Flink run or Flink CDC pipeline to avoid converting bigint unsigned values to decimal?

### DISCUSSION THREAD

**Discussion by songsh2** [apache/flink-cdc#3813] [Q&A]
The source field is of type bigint unsigned, but when it is written to Paimon by MySQL CDC, it is automatically converted to decimal(20, 0).

I added `--mysql_conf bigint.unsigned.handling.mode=long` to the Flink run statement, but it did not work.
If I added `bigint.unsigned.handling.mode: long` to Flink CDC Pipeline yaml，it did not start.

Reference: [GitHub Discussion](https://github.com/apache/flink-cdc/issues/798)

**Comment by Pluto201** [comment]
what version are you using ?

**Reply by songsh2** [reply]
flink 1.8
flinkcdc 3.1
